homeassistant/components/foscam/camera.py

```diff
@@ -65,6 +66,7 @@
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
         vol.Optional(CONF_RTSP_PORT): cv.port,
+        vol.Optional(CONF_STREAM): cv.string,
```

Comment:
We no longer allow changes to YAML configuration for integration that communicate with a device or service.

For more information see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

This integration needs to be adjusted to use a configuration flow. To get started on that, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>